dfttk/aggregate_extraction.py

```diff
@@ -150,3 +156,194 @@ def recursive_extract_configuration_data(
             print(f"Error in {config_dir}: {e}")
     df = pd.concat(df_list, ignore_index=True)
     return df
+
+def extract_convergence_data(path: str) -> pd.DataFrame:
+    """Calculates the energy convergence with respect to ENCUT and plots the results.
```

Comment:
Calculates the energy convergence with respect to ENCUT and kpoints and plots the results.